sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_daily_summary_v1/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+---
+friendly_name: FxA Users Services Daily Summary
+description: |
+  Usage aggregations summary per day
```

Comment:
If is an aggregate the naming for consistency would be `fxa_users_services_daily_aggregates_v1`

Reply:
```ValueError: Invalid task name firefox_accounts_derived__fxa_users_services_daily_aggregates__v1. The task name has to be 1 to 62 characters long.```

😢